### ISSUE TITLE
node-gyp build silently ignored, although this breaks this module

### ISSUE THREAD

**Issue by triccardi-systran** [christkv/kerberos#17]
In https://github.com/christkv/kerberos/blob/52e9fa6a4d39e0fff15e6dbf85328356090f56ba/package.json#L22

```javascript
{
  "scripts": {
    "install": "(node-gyp rebuild 2> builderror.log) || (exit 0)"
  }
}
```

If a build error happens, it's silently ignored.
`mongodb-core` has `kerberos` as `optionalDependencies`, so kerberos should explicitly fail, and let npm optionalDependencies work correctly (ie it's npm that ignores the error, not kerberos itself, shouldn't it?

**Comment by zbjornson** [comment]
Highly agree!

**Comment by panuhorsmalahti** [comment]
:+1:, agree. #31 will fix this.

**Comment by christkv** [comment]
merged #31
